### DISCUSSION TITLE
Our First Hardware Contribution

### DISCUSSION THREAD

**Discussion by zivy** [IBEXImagingCommunity/ibex_imaging_knowledge_base#192] [Announcements]
You read the [original IBEX protocol](https://doi.org/10.1038/s41596-021-00644-9) and want to use the method in your research. Full of excitement you start ordering the hardware components only to find out that some have been discontinued. What to do?

With current technology, you may just be able to design and 3D print the missing hardware. That's exactly what Ed Allen and Ryan Moriarty from the [Thomas Hope Lab](https://labs.feinberg.northwestern.edu/hope/) at Northwestern University did. 

As part of our community's spirit of open science, they not only solved the problem for themselves, they shared their [design and construction instructions for a 3D chamber](https://3d.nih.gov/entries/3DPX-021250) on the [NIH 3D](https://3d.nih.gov).

Thank you Ed and Ryan!

If you are building custom hardware or designing 3D printed hardware components, share them with the community. Our preferred repository for sharing 3D models is the [NIH 3D](https://3d.nih.gov) which provides you with a Digital Object Identifier (DOI), enabling others to cite your work.

For those interested in learning more about the open hardware movement in science, see the Gathering for Open Science Hardware ([GOSH](https://openhardware.science/)) community.

**Comment by radtkea** [comment]
AWESOME!!! Great work Ed and Ryan!!!
